pkg/tests/tasks/security/certificate/cert_manager_test.go

```diff
@@ -15,14 +17,17 @@ import (
 
 func TestCertManager(t *testing.T) {
 	test.NewTest(t).Id("T38").Groups(test.Full, test.ARM, test.InterOp).Run(func(t test.TestHelper) {
-		ns := "foo"
```

Comment:
I like that you introduced `ns.Foo` and the others, but I think keeping this `ns` variable here (and setting it to `ns.Foo`) has the benefit of not having "ns.Foo" strewn all across this test. Also, if one sees `ns.Foo` in the test, they might start wondering whether other parts of the test use a different namespace, whereas if you see just `ns`, you can be pretty sure that this is the only namespace used across the entire test.

Reply:
> Also, if one sees ns.Foo in the test, they might start wondering whether other parts of the test use a different namespace, whereas if you see just ns, you can be pretty sure that this is the only namespace used across the entire test.

I totally don't get this assumption. I understand that you may not like some names or conventions I introduce, but please take into account that you rewrote almost entire project from scratch. This is my first test and I have different opinion and feeling about this code base. It does not make sense for me to reassign `ns.Foo` to `ns`, and using both `ns` and `ns.Bar` is even more complicated. `ns` does not say anything and when I look at cluster resource I don't see namespace named "ns", so it's not better than `ns.Foo` for me.